docs/src/python/nn/layers.rst

```diff
@@ -22,6 +22,10 @@ Layers
    QuantizedLinear
    Conv1d
    Conv2d
+   MaxPooling1d
+   MaxPooling2d
+   AvgPooling1d
+   AvgPooling2d
```

Comment:
If we want to follow the PyTorch naming convention you should drop the `ing` part of the name.